### DISCUSSION TITLE
Add Option to Change Background Color of Plot Area Within Graphs in Android Applications

### DISCUSSION THREAD

**Discussion by labqui** [patrykandpatrick/vico#772] [Ideas]
Hello,

I would like to request a feature that allows developers to change the background color of the plot area within graphs. This customization would enhance visual options and streamline integration of graphs into various designs and themes. I greatly appreciate the excellent graph library that provides such a robust visual experience!

![image (1)](https://github.com/patrykandpatrick/vico/assets/18506384/ee9ecd81-0880-4b6d-8273-72022cbbfed1)

**Comment by Gowsky** [comment]
Hello! You can already do this using the [`Decoration`](https://patrykandpatrick.com/vico/api/2.0.0-alpha.21/core/com.patrykandpatrick.vico.core.cartesian.decoration/-decoration/) API. Since it’s very simple to implement using the API, I’ll be closing the discussion.
